README.md

```diff
@@ -1,17 +1,121 @@
-# jujulib
 
-This project provides a JavaScript API client for interacting with the Juju
+# JS Jujulib
+
+This project provides a JavaScript API client library for interacting with the Juju
 WebSocket API.
```

Comment:
It would be nice to provide links to docs or existance of the API for readers to follow if interested.

Reply:
Good call, I'll update this to include the link directly to the client api once it's live (I don't know what the actual url will be until then) For now they can click through to the link below for the full api.